website/blog/templates/blog/blog_details.html

```diff
@@ -0,0 +1,639 @@
+{% extends 'base.html' %} {% block content %} {% load staticfiles %}{% load widget_tweaks %}
+ <!-- CSS/JS IMPORTS  -->
+<link rel="stylesheet" href="{% static 'css/detail.css' %}">
+<link rel="stylesheet" href="{% static 'css/tag.css' %}">
+<link rel="stylesheet" href="{% static 'css/gh-buttons.css' %}"> {% load static %}
+<script src="{% static 'app.js' %}"></script>
+ <!-- CSS/JS IMPORTS ENDS -->
+
+ <!-- PAGE CONTENT START -->
+<div class="container">
+    {% csrf_token %}
+    <!-- BACK BUTTON -->
+    <small><a href="{% url 'blog:list_blogs' %}" class="text-muted"><strong>< Back to Blogs</strong></a></small>
```

Comment:
Remove the strong tag, so that the hyperlink appear same as it appear in other pages.